tests/main/user-session-env/task.yaml

```diff
@@ -0,0 +1,84 @@
+summary: verify that user environment settings are added
+
+description: |
+    User environment variables are added via /etc/profile.d/snapd.sh (bash/sh
+    specific) or via /lib/environment.d/ helpers. Make sure that at least one of
+    the mechanisms works and XDG_DATA_DIRS and PATH are updated accordingly
+    inside the user session, no matter the shell they use.
+
+# environment generators were introduced in systemd 233, anything before that
+# will not work
```

Comment:
Is it still a concern? (afaiu it's disabled on core for different reason).

Reply:
Ah, I see it's handled below (with a more telling comment), so perhaps this comment should be removed as it's slightly confusing?